components/derive/src/exonum_interface.rs

```diff
@@ -306,21 +306,21 @@ impl ExonumService {
 
             let method = quote! {
                 fn #name(&self, context: Ctx, arg: #arg_type) -> Self::Output {
-                    #cr::runtime::rust::GenericCall::generic_call(
+                    #cr::GenericCall::generic_call(
                         self,
                         context,
                         #descriptor,
-                        #cr::merkledb::BinaryValue::into_bytes(arg),
+                        exonum_merkledb::BinaryValue::into_bytes(arg),
```

Comment:
This forces users to explicitly export `exonum_merkledb` crate, although it is already guaranteed to be reexported by `exonum`. For this reason, `exonum::merkledb` could be better.